Mathlib/CategoryTheory/Monoidal/Internal/FunctorCategory.lean

```diff
@@ -41,106 +42,68 @@ namespace MonFunctorCategoryEquivalence
 
 variable {C D}
 
--- Porting note: the `obj` field of `functor : Mon_ (C ⥤ D) ⥤ C ⥤ Mon_ D` defined below
--- had to be defined separately as `Functor.obj` in order to speed up the compilation
 /-- A monoid object in a functor category induces a functor to the category of monoid objects. -/
 @[simps]
-def Functor.obj (A : Mon_ (C ⥤ D)) : C ⥤ Mon_ D where
+private def functor_obj' (A : Mon_ (C ⥤ D)) : C ⥤ Mon_ D where
   obj X :=
-    { X := A.X.obj X
-      one := A.one.app X
-      mul := A.mul.app X
-      one_mul := congr_app A.one_mul X
-      mul_one := congr_app A.mul_one X
-      mul_assoc := congr_app A.mul_assoc X }
+  { X := A.X.obj X
+    one := A.one.app X
+    mul := A.mul.app X
+    one_mul := congr_app A.one_mul X
+    mul_one := congr_app A.mul_one X
+    mul_assoc := congr_app A.mul_assoc X }
   map f :=
-    { hom := A.X.map f
-      one_hom := by rw [← A.one.naturality, tensorUnit_map]; dsimp; rw [Category.id_comp]
-      mul_hom := by dsimp; rw [← A.mul.naturality, tensorObj_map] }
+  { hom := A.X.map f
+    one_hom := by rw [← A.one.naturality, tensorUnit_map]; dsimp; rw [Category.id_comp]
+    mul_hom := by dsimp; rw [← A.mul.naturality, tensorObj_map] }
   map_id X := by ext; dsimp; rw [CategoryTheory.Functor.map_id]
   map_comp f g := by ext; dsimp; rw [Functor.map_comp]
 
 /-- Functor translating a monoid object in a functor category
 to a functor into the category of monoid objects.
 -/
 @[simps]
-def functor : Mon_ (C ⥤ D) ⥤ C ⥤ Mon_ D where
-  obj := Functor.obj
+private def functor : Mon_ (C ⥤ D) ⥤ C ⥤ Mon_ D where
+  obj := functor_obj'
   map f :=
-    { app := fun X =>
-        { hom := f.hom.app X
-          one_hom := congr_app f.one_hom X
-          mul_hom := congr_app f.mul_hom X } }
-set_option linter.uppercaseLean3 false in
-#align category_theory.monoidal.Mon_functor_category_equivalence.functor CategoryTheory.Monoidal.MonFunctorCategoryEquivalence.functor
+  { app := fun X =>
+    { hom := f.hom.app X
+      one_hom := congr_app f.one_hom X
+      mul_hom := congr_app f.mul_hom X } }
 
--- Porting note: the `obj` field of `inverse : (C ⥤ Mon_ D) ⥤ Mon_ (C ⥤ D)` defined below
--- had to be defined separately as `Inverse.obj` in order to speed up the compilation
 /-- A functor to the category of monoid objects can be translated as a monoid object
 in the functor category. -/
 @[simps]
-def Inverse.obj (F : C ⥤ Mon_ D) : Mon_ (C ⥤ D) where
+private def inverse_obj' (F : C ⥤ Mon_ D) : Mon_ (C ⥤ D) where
```

Comment:
I do not understand why these definitions are `private`.

Reply:
I had been thinking these would never need to be accessed once the equivalence was built, but I agree this is not helpful, and the namespace is already sufficient to indicate these are "implementation details".

Reply:
(Removed now.)